src/bin/ion/commands/generate/generator.rs

```diff
@@ -1,15 +1,24 @@
-use crate::commands::generate::context::{AbstractDataType, CodeGenContext, SequenceType};
-use crate::commands::generate::result::{invalid_abstract_data_type_error, CodeGenResult};
+use crate::commands::generate::context::{CodeGenContext, SequenceType};
+use crate::commands::generate::model::{
+    AbstractDataType, DataModelNode, EnumBuilder, FieldPresence, FieldReference,
+    FullyQualifiedTypeReference, NamespaceNode, ScalarBuilder, SequenceBuilder, StructureBuilder,
+    WrappedScalarBuilder, WrappedSequenceBuilder,
+};
+use crate::commands::generate::result::{
+    invalid_abstract_data_type_error, invalid_abstract_data_type_raw_error, CodeGenResult,
+};
 use crate::commands::generate::templates;
-use crate::commands::generate::utils::{Field, JavaLanguage, Language, NestedType, RustLanguage};
 use crate::commands::generate::utils::{IonSchemaType, Template};
+use crate::commands::generate::utils::{JavaLanguage, Language, RustLanguage};
 use convert_case::{Case, Casing};
+use ion_rs::Value;
```

Comment:
(🗺️ PR tour) This was changed due to dependency change for `ion-schema-rust` which removed reexport of `ion-rs`.